inc/common.class.php

```diff
@@ -140,4 +140,21 @@ public static function recursiveRmdir($dir) {
       }
       return rmdir($dir);
    }
+
+   /**
+    * Get the maximum value of a column for a given itemtype
+    * @param CommonDBTM $item
+    * @param string $condition
+    * @param string $fieldName
+    * @return NULL|integer
+    */
+   public static function getMax(CommonDBTM $item, $condition, $fieldName) {
+      $rows = $item->find($condition, "`$fieldName` DESC", '1');
+      $line = array_pop($rows);
+      if ($line === null) {
+         return null;
+      }
+      return $line[$fieldName];
+   }
```

Comment:
A helper method I recently created for Formcreator. Useful to easily get the highest existing ID in the MQTT logs table (see refactored test s below)